### DISCUSSION TITLE
Scanning for unknown device types

### DISCUSSION THREAD

**Discussion by Fayrewood** [merbanan/rtl_433#2770] [Q&A]
Could you please guide me on limiting my scan of only unknown devices please?  I have a bespoke 433Mhz tracking device that I would like to capture packets from but I can't decipher the CLI options to limit to only unknown devices.

Many thanks!

**Comment by zuckschwerdt** [comment]
If you mean to capture samples then use `-S unknown`
S.a. https://triq.org/rtl_433/ANALYZE.html
Check the samples in https://triq.org/pdv/ and then feed them back with `rtl_433 -A filename.cu8`
